File: 3 - Estrutura de Decisão/Exercicio.26.py
#26. Faça um programa para ler 3 números reais e imprimi-los em ordem crescente. 
# Se os números forem iguais, o cálculo o programa não deve ordená-los.

num1 = float(input("Digite o primeiro número: "))
num2 = float(input("Digite o segundo número: "))
num3 = float(input("Digite o terceiro número: "))

if num1 == num2 and num2 == num3:
    print("Os números são iguais:", num1, num2, num3)
else:
    menor = min(num1, num2, num3)
    maior = max(num1, num2, num3)
    meio = (num1 + num2 + num3) - menor - maior

    print("Os números em ordem crescente são:", menor, meio, maior)

File: Livro - Módulo Java/3 - Estrutura de Decisão/Exercicio.26.java
// Faça um programa para ler 3 números reais e imprimi-los em ordem crescente. Se os números forem iguais, o cálculo o programa não deve ordená-los

import java.util.Scanner;

public class Exercicio26 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o primeiro número: ");
        double n1 = scanner.nextDouble();

        System.out.print("Digite o segundo número: ");
        double n2 = scanner.nextDouble();

        System.out.print("Digite o terceiro número: ");
        double n3 = scanner.nextDouble();

        if (n1 == n2 || n2 == n3) {
            System.out.println("Os números são iguais " + n1 + " , " + n2 + " e " + n3);
        } else {
            double menor = Math.min(Math.min(n1, n2), n3);
            double maior = Math.max(Math.max(n1, n2), n3);
            double meio = (n1 + n2 + n3) - menor - maior;

            System.out.println("Os números em ordem crescente são " + menor + " , " + meio + " e " + maior);
        }

        scanner.close();
    }
}